doc/rst/tools/mason/mason.rst

```diff
@@ -154,6 +159,61 @@ To try out different values at runtime, pass the values for ``number`` to ``maso
    this argument, this command will run the executable over 4 locales.
 
 
+Testing your Project
+====================
+
+Testing a Mason project is easy! Start by adding test files to the /test directory and specifing
```

Comment:
I think it's more conventional to refer to a subdirectory without the leading `/`, e.g. `test/` instead of `/test`.

Also, let's inline code this dir:

``` ``test/`` ```